### PR TITLE
Delete files only after envelope gets completed

### PR DESCRIPTION
**Pull request by rkondratowicz**
https://tools.hmcts.net/jira/browse/BPS-788

### CODE REVIEW

**Comment by rkondratowicz** [comment]
good find @doncem

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/Processor.java [hunk 1]
```diff
@@ -66,7 +67,11 @@ protected void processParsedEnvelopeDocuments(
             return;
         }
 
-        deleteBlob(envelope, cloudBlockBlob);
+        try {
+            cloudBlockBlob.breakLease(0);
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/Processor.java]: Any reason we are breaking the lease here ? Worth adding a comment regarding the same.

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/Processor.java]: This part was being done in (now deleted) `deleteBlob(...)` method.
Keeping it as it was.

**Comment by rkondratowicz** [comment]
Closing, going with alternative solution from https://github.com/hmcts/bulk-scan-processor/pull/815

**Pull request closed without merging**